### PR TITLE
Post repo-merge doc updates

### PR DESCRIPTION
**Pull request by bsweger**
Update of the docs after the great repo merge of April, 2016:

* Move background info from dataactcore README to main README
* Remove vestiges of old install directions in favor of the consolidated version
* Install direction updates/fixes
* Add background blurb and link to main README in each of the individual READMES (core, api, validator)
* Replace "coming soon" with a link to the alpha website, hooray!

### CODE REVIEW

File: dataactcore/README.md [hunk 1]
```diff
@@ -6,78 +6,19 @@ The DATA Act Core is a collection of common components used by other DATA Act re
...
 * `staging` - Holds records that passed validation. Each file validated will have a table in this database, named based on the job ID. If the `file_status` table in the `error_data` database lists the file as completed, each record in the input file will be present in either this staging table or the error report.
 * `user_manager` - Holds a mapping between user names and user IDs to be used for providing submission history information to a user.
 * `validation` - Contains all the information a submitted file is validated against. The `file_columns` table details what columns are expected in each file type, and the rule table maps all defined single-field rules to one of the columns specified in `file_columns`. The `multi_field_rule` table stores rules that involve a set of fields, but are still checked against a single record at a time. Finally, the `tas_lookup` table holds the set of valid TAS combinations, taken from the TAS csv file discussed in the setup section.
 
-### How It Works
+## DATA Act Core Project Layout
 
 The DATA Act Core repository is a collection of common components used by other
```
Comment (jworcestBAH) [inline dataactcore/README.md]: We should look through this and other docs for uses of "repository" that are no longer correct.  We could change it to "package".

Comment (bsweger) [inline dataactcore/README.md]: @jworcestBAH good point--pushed a commit to address this

File: doc/CONTRIBUTING.md [hunk 2]
```diff
@@ -164,14 +162,14 @@ Before running the broker, you'll need to provide a few configuration options. U
         * `password`
     * under _logging_:
         * `log_files`
-5. Save your changes to config.yml
+4. Save your changes to config.yml
```
Comment (jworcestBAH) [inline doc/CONTRIBUTING.md]: This is clearly an improvement on the line replaced.

Comment (bsweger) [inline doc/CONTRIBUTING.md]: Thanks, I thought so too!

**Comment by jworcestBAH** [comment]
Looks good, merging

**Pull request merged**